src/directory/directory.js

```diff
@@ -464,7 +464,7 @@ const directory = {
             filters: ["js"],
           },
           {
-            title: "Create In-App Messaging campaigns",
+            title: "Create an In-App Messaging campaign",
```

Comment:
Nit: Should the route be updated to match?

QQ I'm fuzzy on the content but isn't it ok to leave it plural? Can't they make as many as they want or do our instructions pointedly guide them through making just one? Just wondering what necessitated this change